src/osemgrep/reporting/Matches_report.ml

```diff
@@ -29,6 +29,27 @@ let group_titles = function
   | `Nonblocking -> "Non-blocking Code Finding"
   | `Blocking -> "Blocking Code Finding"
   | `Merged -> "Code Finding"
+  | `Valid -> "Valid Secrets Finding"
```

Comment:
we probably define a proper type for this.
The orignal code was using undefined polymorphic variants but would probably
be better to define a proper type. We could even reuse the variants in semgrep_output_v1.atd for the valid/invalid stuff and do a union as in type groups = [ ... | Semgrep_output_v1_t.validity ]